src/components/UserPortal/ChatRoom/ChatRoom.tsx

```
@@ -53,12 +53,7 @@ interface IChatRoomProps {
 }
 
 export default function chatRoom(props: IChatRoomProps): JSX.Element {
-  const { t } = useTranslation('translation', {
-    keyPrefix: 'userChatRoom',
-  });
-  const { t: tErrors } = useTranslation('translation', {
-    keyPrefix: 'userChatRoom.errorBoundary',
-  });
+  const { t } = useTranslation('translation');
```

Biome (2.4.6)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)